### PR TITLE
Updated JLL version used by SymEngine.jl and bumped package version

### LINKED ISSUE #251: Update SymEngine_jll
Hi!

Currently, the version of SymEngine_jll is based on SymEngine v0.8.1+0, which was published in Nov 2021. The latest version of SymEngine supports serialization, but Julia-wrapper can't use it because of an old version of jll. Could you update it?

This will fix CI check for commit #246. Also, after updating jll I could add another PR for serialization/deserialization which will potentially fix #242 and #198 and allow the use of SymEngine for multiprocessing and distributed computing.

### PR DESCRIPTION
**Pull request by PerformanceCoder**
Pull request to update JLL-version used by SymEngine.jl and SymEngine.jl package version. Fixes #251

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ SymEngine_jll = "3428059b-622b-5399-b16f-d347a77089a4"
 Compat = "0.63.0, 1, 2, 3"
 RecipesBase = "0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 1.0"
 SpecialFunctions = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 1, 2"
-SymEngine_jll = "0.8"
+SymEngine_jll = "0.9"
 julia = "1.3"
```
Comment (isuruf) [inline Project.toml:18]: Need to update this to `1.6` and then update the CI too

Comment (PerformanceCoder) [reply Project.toml]: Updated this PR and created #253 for CI

Comment (isuruf) [reply Project.toml]: Thanks. Can you rebase on top of master now?

File: Project.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name = "SymEngine"
 uuid = "123dc426-2d89-5057-bbad-38513e3affd8"
-version = "0.8.7"
+version = "0.9"
```
Comment (isuruf) [inline Project.toml:3]: Can you keep this at 0.8.7? I'll bump this when we are ready to release.

**Comment by isuruf** [comment]
Thanks

File: Project.toml (hunk 2 above)
Comment (PerformanceCoder) [reply Project.toml]: Fixed

**Pull request merged**